time-elements.js

```diff
@@ -561,7 +563,9 @@
     var options = {
       hour: el.getAttribute('hour'),
       minute: el.getAttribute('minute'),
-      second: el.getAttribute('second')
+      second: el.getAttribute('second'),
+      timeZone: el.getAttribute('timeZone'),
+      timeZoneName: el.getAttribute('timeZoneName')
```

Comment:
I don't get the purpose of these new attributes. Why would you set a `timezone=PST` value on the server? Wasn't the point of this pull requests to simply opt into showing the local timezone information to the current user, so that they are certain that the timestamp they're seeing is in their local timezone?